src/components/structures/MatrixChat.tsx

```diff
@@ -1202,9 +1202,9 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
         const roomLink = makeRoomPermalink(roomId);
         const success = await copyPlaintext(roomLink);
         if (!success) {
-            Modal.createTrackedDialog("Unable to copy room", "", ErrorDialog, {
-                title: _t("Unable to copy room"),
-                description: _t("Unable to copy the room to the clipboard."),
+            Modal.createTrackedDialog("Unable to copy room link", "", ErrorDialog, {
```

Comment:
the first 2 params here are just for Analytics (Matomo) that we don't really get any useful data out of so don't matter very much btw